compiler/src/Language/Bond/Codegen/Java/Util.hs

```diff
@@ -66,3 +69,9 @@ isGenericBondStructType _ = False
 ifThenElse :: Bool -> Text -> Text -> Text
 ifThenElse True thenCondition _ = thenCondition
 ifThenElse False _ elseCondition = elseCondition
+
+-- takes a bit count and a number and returns its two's complement
+twosComplement :: Integer -> Integer -> Integer
```

Comment:
`twosComplement` explains the what, but not the why, which is more important. I'd rather you called this `asSigned` or something like that.